runtime/compiler/codegen/J9CodeGenerator.cpp

```diff
@@ -262,6 +262,13 @@ J9::CodeGenerator::fastpathAcmpHelper(TR::Node *node, TR::TreeTop *tt, const boo
    TR::Block* targetBlock = prevBlock->splitPostGRA(tt->getNextTreeTop(), cfg, true, NULL);
```

Comment:
Is this something splitPostGRA should be doing?

Reply:
Do you mean moving the code that copies live local info into `splitPostGRA()`? 
I think we could add `prevBlock` as a default parameter to `splitPostGRA()` [1]. But`split()` also needs to be updated because both the new EBB and the new target block need to have the live local info copied over from the `prevBlock`. 

[1]
```c++
TR::Block * OMR::Block::split(..., TR::Block * prevBlock = NULL)
TR::Block * OMR::Block::splitPostGRA(..., TR::Block * prevBlock = NULL)
```

Reply:
No need to pass `*prevBlock`, a boolean to indicate if copying live local info should be good enough. 
```c++
TR::Block * OMR::Block::split(..., bool isCopyLiveLocals = false)
TR::Block * OMR::Block::splitPostGRA(..., bool isCopyLiveLocals = false)
```

Reply:
per offline discussion, I feel that it is incorrect for splitPostGRA to no copy the live local information. Further, because ::split should not be used after GRA there is no reason it should support copying the live local information. The use of ::split post-GRA is deprecated and we should clean up the uses to use the splitPostGRA API in due course. For now, as a tactical fix, where ::split is used after GRA we can add the manual copy of the live local information until we can switch the callsite to use splitPostGRA.

Reply:
Having splitPostGRA copying the liveness info would fix other places that call splitPostGRA, such as in jprofiler. However, jprofiler also uses ::split in some places. Once we fix splitPostGRA, we should replace split used after GRA.